bluesky/plan_patterns.py

```diff
@@ -269,6 +269,90 @@ def spiral_fermat(x_motor, y_motor, x_start, y_start, x_range, y_range, dr,
     return cyc
 
 
+def inner_list_product(args):
+    '''Scan over one multi-motor trajectory.
+
+    Parameters
+    ----------
+    args : list of {Positioner, position_list}
```

Comment:
`of {Positioner, position_list}` gives me the impression of a nested structure. Would it be better to just say `list`? I'm sure mileage will very here, and I could be in the minority.

Reply:
fixed